### PR TITLE
libwbxml: init at 0.11.7

### PR DESCRIPTION
**Pull request by markus-heinrich**
###### Motivation for this change
Add the WBXML Library.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -7952,6 +7952,12 @@
     githubId = 668926;
     name = "Maximilian Güntner";
   };
+  mh = {
```
Comment (bobby285271) [inline maintainers/maintainer-list.nix:7955]: Please put the `maintainers/maintainer-list.nix` edit in a separate commit with commit message `maintainers: add mh`. (So there are exactly 2 commits in this PR).

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -7952,6 +7952,12 @@
...
+    email = "68288772+markus-heinrich@users.noreply.github.com ";
```
Comment (bobby285271) [inline maintainers/maintainer-list.nix:7956]:
```suggestion
    email = "68288772+markus-heinrich@users.noreply.github.com";
```

File: pkgs/development/libraries/libwbxml/default.nix [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, lib, cmake, expat }:
+
+let
+in stdenv.mkDerivation rec {
```
Comment (bobby285271) [inline pkgs/development/libraries/libwbxml/default.nix:4]:
```suggestion
stdenv.mkDerivation rec {
```

File: pkgs/development/libraries/libwbxml/default.nix [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    owner = "libwbxml";
+    repo = "libwbxml";
+    rev = "${pname}-${version}";
+    sha256 = "sha256:1b81rbkd28d9059vh8n5gql73crp8h7av67kkmr6lhicl08fv2xx";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ expat ];
+
+  patchPhase = ''
```
Comment (bobby285271) [inline pkgs/development/libraries/libwbxml/default.nix:18]:
```suggestion
  postPatch = ''
```

File: pkgs/development/libraries/libwbxml/default.nix [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    sed -i 's/^SET.*$//' ../source/cmake/CMakeLists.txt
```
Comment (bobby285271) [inline pkgs/development/libraries/libwbxml/default.nix:19]:
```suggestion
    sed -i 's/^SET.*$//' cmake/CMakeLists.txt
```

File: pkgs/development/libraries/libwbxml/default.nix [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/libwbxml/libwbxml";
+    description = "The WBXML Library (aka libwbxml) contains a library and its associated tools to Parse, Encode and Handle WBXML documents";
+    maintainers = with maintainers; [ mh ];
+    platforms = platforms.linux;
+    license = licenses.lgpl21;
```
Comment (bobby285271) [inline pkgs/development/libraries/libwbxml/default.nix:27]:
```suggestion
    license = licenses.lgpl21Plus;
```

**Comment by markus-heinrich** [comment]
Thanks for the feedback. I applied your suggestions.

**Pull request merged**